spec/deepl_translate_spec.rb

```diff
@@ -6,15 +6,16 @@
 
 RSpec.describe 'DeepL Translation' do
   nil_value_test = ['nil-value-key', nil, nil]
-  text_test      = ['key', "Hello, %{user} O'Neill! How are you?", "¡Hola, %{user} O'Neill! ¿Cómo estás?"]
+  text_test      = ['key', "Hello, %{user} O'Neill! How are you?", "¡Hola, %{user} O'Neill! ¿Qué tal estás?"]
```

Comment:
Not sure why this changes :/ and Spanish is not my forté, I googled it and it seems like it might be right.

Reply:
Well that's definitely not right but that's OK it's just a test.

Reply:
Yeah, I guess it is the users problem when using interpolation :/